lib/provider/camunda/parts/VariableMappingProps.js

```diff
@@ -313,13 +318,13 @@ module.exports = function(group, element, bpmnFactory, translate) {
       if (mapping) {
         if (!values.source) {
           validation.source =
-            'Mapping must have a ' + (values.sourceLabel.toLowerCase() || 'value');
+            translate('Mapping must have a') + ' ' + (values.sourceLabel.toLowerCase() || 'value');
```

Comment:
```suggestion
          validation.source = values.sourceLabel ?
            translate('Mapping must have a {value}', { value: values.sourceLabel.toLowerCase() }) :
            translate('Mapping must have a value');
```
Unfortunately, for now there is no easy way to add default values, but we should do it this way.

Reply:
It seems like this suggestion block does not apply for multilines 🙄.

Reply:
ok.  Just wanted to have it consistent over the component.
In many similar places, the message is just "Must provide a value", with no dot.

Can we just omit the dot?

Reply:
We can think about it in a separate PR, as this is simply not a part of i18n, ok?